console-backend/src/info.rs

```diff
@@ -10,3 +10,13 @@ pub async fn get_info(endpoint_source: web::Data<EndpointSourceType>) -> impl Re
         Err(err) => HttpResponse::InternalServerError().json(json!( {"error": err.to_string()})),
     }
 }
+
+#[get("/endpoints")]
```

Comment:
```suggestion
#[get("/drogue-endpoints")]
```

Taking a look at how others do it -> https://en.wikipedia.org/wiki/List_of_/.well-known/_services_offered_by_webservers

Having just "endpoints" would work as well … but I do prefer when others prefix that with their technology or project. So I would suggest to use `drogue-endpoints`.